src/main/java/com/swervedrivespecialties/exampleswerve/RobotContainer.java

```diff
@@ -0,0 +1,70 @@
+package com.swervedrivespecialties.autonrefresh;
+
+import com.swervedrivespecialties.autonrefresh.subsystems.DrivetrainSubsystem;
+import com.swervedrivespecialties.autonrefresh.util.AutonChooser;
+import com.swervedrivespecialties.autonrefresh.util.BeakXboxController;
+import com.swervedrivespecialties.autonrefresh.util.util;
+import com.swervedrivespecialties.exampleswerve.commands.DriveCommand;
+
+public class RobotContainer {
+
+    public DrivetrainSubsystem drive = new DrivetrainSubsystem.getinstance(); 
+
+    private BeakXboxController controller1 = new BeakXboxController(RobotMap.PRIMARY_CONTROLLER_USB_PORT);
+    private BeakXboxController controller2 = new BeakXboxController(RobotMap.SECONDARY_CONTROLLER_USB_PORT);
+    private BeakXboxController controller3 = new BeakXboxController(RobotMap.TERTIARY_CONTROLLER_USB_PORT);
+
+    AutonChooser ac = AutonChooser.getinstance();
```

Comment:
```suggestion
```

None of this code will work yet because you haven't written any of the underlying classes